static/app/components/sidebar/index.tsx

```diff
@@ -117,7 +117,17 @@ function Sidebar() {
   const horizontal = useMedia(`(max-width: ${theme.breakpoints.medium})`);
 
   // Avoid showing superuser UI on self-hosted instances
-  const hasSuperuserSession = isActiveSuperuser() && !ConfigStore.get('isSelfHosted');
+  const hasSuperuserSession = () => {
+    return isActiveSuperuser() && !ConfigStore.get('isSelfHosted');
+  };
+
+  // Avoid showing superuser UI on certain organizations
+  const isExcludedOrg = () => {
+    const shouldExclude = HookStore.get('component:superuser-warning-excluded')[0]?.(
```

Comment:
@cathteng  I will change the the `isSuperuser={hasSuperuserSession() && !isExcludedOrg()}>`, but the `isSuperuser` prop is also not just for the sidebar, it is also a condition to render the banner at the top. maybe `showSuperuserWarningBanner` might be better?

Reply:
hmm i could be wrong i'm not sure if it matters where the `{hasSuperuserSession && <Hook name="component:superuser-warning" />}` lives, since it's rendering the warning in a very specific spot. `showSuperuserWarning` should be enough though it we just want to change the prop name